examples/rules_ios/xcodeproj_targets.bzl

```
@@ -25,3 +26,25 @@ IOS_BUNDLE_ID = "rules-xcodeproj.example"
 TEAMID = "V82V4GQZXM"
 
 SCHEME_AUTOGENERATION_MODE = "all"
+
+def get_xcode_schemes():
+    return [
+        xcode_schemes.scheme(
+            name = "iOSAppUnitTests_Scheme",
+            test_action = xcode_schemes.test_action(
+                env = {
+                    "IOSAPPSWIFTUNITTESTS_CUSTOMSCHEMEVAR": "TRUE",
+                },
+                targets = [
+                    "//iOSApp/Test/ObjCUnitTests:iOSAppObjCUnitTests",
+                ],
+                post_actions = [
+                    xcode_schemes.pre_post_action(
+                        name = "Run After Tests",
+                        script = "echo \"Hi\"",
+                        expand_variables_based_on = "//iOSApp/Test/ObjCUnitTests:iOSAppObjCUnitTests",
+                    ),
+                ],
+            ),
+        ),
+    ]
```

Any reason to not just match `XCODEPROJ_TARGETS` and avoid the function?
```suggestion
XCODEPROJ_SCHEMES = [
    xcode_schemes.scheme(
        name = "iOSAppUnitTests_Scheme",
        test_action = xcode_schemes.test_action(
            env = {
                "IOSAPPSWIFTUNITTESTS_CUSTOMSCHEMEVAR": "TRUE",
            },
            targets = [
                "//iOSApp/Test/ObjCUnitTests:iOSAppObjCUnitTests",
            ],
            post_actions = [
                xcode_schemes.pre_post_action(
                    name = "Run After Tests",
                    script = "echo \"Hi\"",
                    expand_variables_based_on = "//iOSApp/Test/ObjCUnitTests:iOSAppObjCUnitTests",
                ),
            ],
        ),
    ),
]
```

---

Can't store the schemes in variables (covered in the FAQ).

---

Yeah copied the same pattern from the other examples too

---

Well then ignore my unhelpful suggestions 😆